### ISSUE TITLE
Docker Compose Lines

### ISSUE THREAD

**Issue by rrbebeau** [prakhar1989/docker-curriculum#365]
I think the newest versions of Docker use the command: ```docker compose``` instead of ```docker-compose```.

I think making a note of that in the tutorial would be helpful!

**Comment by lenasolarova** [comment]
Agreed, I had slight issues with finding the version installed at first before I found this https://docs.docker.com/reference/cli/docker/compose/version/

Basically using `docker compose version` instead of `docker-compose --version`

**Comment by alexkrz** [comment]
Also supporting this issue.
`docker compose` has been changed from a Python implementation to a Go implemenation as documented here:
<https://docs.docker.com/compose/intro/history>

**Comment by prakhar1989** [comment]
Thank you for raising this! The tutorial documentation and Dockerfile guide have been updated to modern syntax and standards in https://github.com/prakhar1989/docker-curriculum/commit/1ac2bde324e2ba36b7f529140ec35591713be01d. Closing this issue now.
